.gitignore

```diff
@@ -7,3 +7,4 @@ Carthage
 Pods
 .build/
 build/
+Pods/
```

Comment:
Isn't this included with the `Pods` in line 7? 🤷‍♂️

Reply:
oh, this is due to a auto merge with main. I will remove this line